### PR TITLE
[civ2][gpu/4] migrate rllib multi-gpu tests to civ2

### PR DESCRIPTION
**Pull request by can-anyscale**
Migrate rllib multi-gpu tests to civ2. Need to mark on test as flaky so unfortunately no cost improvements.

Test:
- CI

### CODE REVIEW

**Review by aslonnie — COMMENTED** [review]
(still in draft, and has conflicts)

File: .buildkite/rllib.rayci.yml [hunk 1]
```diff
@@ -126,6 +126,30 @@ steps:
     depends_on: rllibbuild
     job_env: forge
 
+  - label: ":brain: rllib: multi-gpu tests"
+    tags: rllib
```
Comment (aslonnie) [inline .buildkite/rllib.rayci.yml:130]: could you also add the gpu tag?

Comment (can-anyscale) [reply .buildkite/rllib.rayci.yml:130]: ah yes

**Comment by can-anyscale** [comment]
added gpu tags

**Review by aslonnie — APPROVED** [review]

**Pull request merged**